Cargo.toml

```diff
@@ -78,13 +78,13 @@ tracing = { version = "0.1.41", default-features = false }
 
 # Testing
 pprof = "0.14.0"
-proptest = "1.5.0"
+proptest = "1.6.0"
 criterion = "0.5.1"
 
 # Serialization
 rkyv = "0.8.9"
-serde = { version = "1.0.215", default-features = false }
-serde_json = { version = "1.0.133", default-features = false }
+serde = { version = "1.0.217", default-features = false }
+serde_json = { version = "1.0.135", default-features = false }
```

Comment:
```suggestion
serde = { version = "1.0", default-features = false }
serde_json = { version = "1.0", default-features = false }
```

Reply:
if we don't absolutely need those minimum versions, then don't specify them. makes it easier to get into dependency hell.